src/pages/admin/employees/[id].tsx

```diff
@@ -1,6 +1,6 @@
 import AdminHeader from "@/components/AdminHeader";
 import MyHead from "@/components/MyHead";
-import Show from "@/components/Show";
+import EmployeeShow from "@/components/UserShow";
```

Comment:
名前を変えてインポートすると後で混乱しやすいので、なるべく同じ名前でインポートすることをおすすめします。

```suggestion
import UserShow from "@/components/UserShow";
```

Reply:
https://github.com/tatata05/next-app/pull/10/commits/349245a1dc1bc42cb457728f164d5cf3502e4366
上記コメントのコミットと同じものですが、修正しております。